### PR TITLE
Increased timeout to prevent build abortion (needs RCA)

### PR DESCRIPTION
**Pull request by LinkMJB**
#### What does this PR do?
Increases timeout to prevent build abortion (needs RCA)

#### Who is reviewing it? 
@TonyMorrison 
@shaundmorris 

#### Select relevant component teams: 
@codice/build 

#### Ask 2 committers to review/merge the PR and tag them here.
@mcalcote
@shaundmorris

#### How should this be tested?
N/A

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -98,7 +98,7 @@ pipeline {
                 stage ('Linux') {
                     steps {
                         // This timeout was reduced from 3 hours after this stage was modified to no longer run the itests. In the future, we may want to reduce it further if there is confidence that the build will finish faster.
```
Comment (shaundmorris) [inline Jenkinsfile]: ❔ should we update the comment? It isn't reduced from 3 hours anymore.

Comment (LinkMJB) [reply Jenkinsfile]: @shaundmorris sounds good, I'll remove the comment

Comment (LinkMJB) [reply Jenkinsfile]: @shaundmorris all set

**Comment by LinkMJB** [comment]
build now

**Pull request merged**